package.json

```diff
@@ -1,20 +1,45 @@
 {
+  "name": "content-tag",
+  "description": "A rust program that uses a fork of swc to parse and transform Javascript containing the content-tag proposal",
+  "license": "MIT",
   "version": "1.1.2",
+  "type": "module",
   "repository": {
     "type": "git",
     "url": "git@github.com:embroider-build/content-tag.git"
   },
+  "files": [
+    "pkg"
+  ],
+  "exports": {
+    ".": {
+      "types": "index.d.ts",
+      "browser": {
```

Comment:
this conditional export is what enables browser mode

Reply:
(assuming vite isn't falling back to the "default" exports (pkg/node/content_tag.cjs) and polyfilling all the node stuff for the browser)